.github/CODEOWNERS

```diff
@@ -1 +1,40 @@
+# File extension owners
+
+*.sol @yorhodes @asaj @tkporter
```

Comment:
How do these interact with the owners below? I presume union?

Reply:
later overrides earlier